app/main/forms/direct_award_forms.py

```diff
@@ -39,3 +39,29 @@ class DidYouAwardAContractForm(FlaskForm):
             (STILL_ASSESSING, 'We are still assessing services')
         ]
     )
+
+
+class WhyDidYouNotAwardForm(FlaskForm):
+    why_did_you_not_award_the_contract = RadioField(
+        "Why didn't you award the contract?",
+        validators=[InputRequired(message="Please select a reason for why you didn't award the contract")]
```

Comment:
Can we double check with @karlchillmaid but "for why" doesn't seem like good grammar to me - "reason that you didn't" perhaps?